src/components/MediaGallerySection/media-gallery-section.stories.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import MediaGallerySection, { MediaGallerySectionProps } from './index';
+
+export default {
+    title: 'Components/MediaGallerySection',
+    component: MediaGallerySection,
+    argTypes: {
+        type: { table: { disable: true } },
+        colors: {
+            options: ['colors-a', 'colors-b', 'colors-c', 'colors-d', 'colors-e', 'colors-f', 'colors-g', 'colors-h', 'colors-i'],
+            defaultValue: 'colors-d',
+            control: { type: 'select' }
+        },
+        columns: {
+            options: [1, 2, 3, 4, 5, 6],
+            defaultValue: 4,
+            control: { type: 'select' }
+        },
+        backgroundWidth: {
+            options: ['full', 'inset'],
+            defaultValue: 'full',
+            control: { type: 'select' }
+        },
+        elementId: {}
+    }
+};
+
+const Template = (args: MediaGallerySectionProps) => <MediaGallerySection {...args} />;
+
+const args: MediaGallerySectionProps = {
+    title: 'Some photos',
```

Comment:
@smnh @eladroz. These default values should ideally come direct from the model to avoid repetition and de-syncing content. The values shown in the story should be the same as the model defaults and as you can see here the values are already diverging.

@ScottyMJacobson no need to fix this. Just wanted to highlight it to Simon for a possible future improvement.

Reply:
That makes sense! I mean - we could parse the YAML and pass the defaults into the args (or use the same stackbit innards to do it), right? It's all just a JavaScript runtime

_(This goes back to my joking "singularity" comment over slack about how storybook and stackbit studio aren't so far apart after all...)_